docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: '3.4'
```

Usar versão mais recente
```suggestion
version: '3.8'
```